src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -1234,7 +1245,10 @@ namespace RuntimeLanguageProcessor {
 					if ( DeveloperFlag ) gio::write( OutputFileDebug, fmtA ) << "Number=\"" + StringToken + "\"";
 					Token( NumTokens ).Number = ProcessNumber( StringToken, ErrorFlag );
 					if ( DeveloperFlag && ErrorFlag ) gio::write( OutputFileDebug, fmtA ) << "Numeric error flagged";
-					if ( MinusFound ) Token( NumTokens ).Number = -Token( NumTokens ).Number;
+					if ( MinusFound ) {
+						Token( NumTokens ).Number = -Token( NumTokens ).Number;
+						MinusFound = false;
+					}
```

Comment:
I question whether this required now. I believe this was only used for * - found in equations. Not sure it would be used for anything else.